composer.json

```
@@ -18,13 +18,13 @@
     }
   ],
   "require": {
-    "php": "^7.2|^8.0",
+    "php": "^8.1",
     "api-platform/core": "^2.6.4",
-    "symfony/http-client": "^5.4|^6.0",
-    "symfony/property-access": "^5.4|^6.0",
-    "symfony/serializer": "^5.4|^6.0",
-    "symfony/validator": "^5.4|^6.0",
-    "symfony/event-dispatcher": "^5.4|^6.0",
+    "symfony/http-client": ">=6.0 <6.1",
```

```suggestion
    "symfony/http-client": "^6.0",
```

Mayve you can do like that and leverage [conflict](https://getcomposer.org/doc/04-schema.md#conflict)?